### PR TITLE
added styling

### PR DESCRIPTION
**Pull request by babsjohnson**

### CODE REVIEW

**Comment by BTanaka11** [comment]
I'll take this one. First thing tomorrow morning, I'll try a thorough review.

File: client/src/components/Q&A/Answers.jsx [hunk 1]
```diff
@@ -29,20 +39,18 @@ const Answers= ({question}) => {
     .catch((err) => (console.log('GET ANSWER', err)))
   }, [id])
 
-
-
   return (
     <div>
       {answers.slice(0,2).map((answer) => (
```
Comment (BTanaka11) [inline client/src/components/Q&A/Answers.jsx:44]: next time, if you want to get rid of warning of "each child should have a unique key prop", use the pattern:

{answers.slice(0,2).map((answer, **index**) => (
    <A **key={index}**>
    </A>

File: client/src/components/Q&A/QAList.jsx [hunk 2]
```diff
@@ -37,16 +49,15 @@ const QAList = (/*   product_id   */) => {
...
     <div>
       {questions.slice(0, n).map((question) => (
-        <> <span>
-          {/* <label>Q:</label> */}
-          <p>Q: {question.question_body}</p>
-
-          <small> Helpful? <softButton>Yes </softButton>
+      <>
+        <Inline>
+          <p><b><large>Q: &nbsp; {question.question_body}</large></b></p>
```
Comment (BTanaka11) [inline client/src/components/Q&A/QAList.jsx:54]: browser says html element "<large>" isn't valid. If you're trying to change fontsize, try "<big>"

File: client/src/components/Q&A/SearchBar.jsx [hunk 3]
```diff
@@ -1,23 +1,38 @@
...
-    border: none;
-  `
-
   return (
     <div>
-    <form onSubmit={(e) => {
+    <Form onSubmit={(e) => {
       e.preventDefault();
       setSearch(value);
       setValue('');}}>
```
Comment (BTanaka11) [inline client/src/components/Q&A/SearchBar.jsx:28]: I don't think this line is needed. When setSearch changes parent's state, it'll trigger rerender of it and all child components. When Searchbar is rerendered, the useState line will set value to ''.

Comment (BalajiSathiya) [reply client/src/components/Q&A/SearchBar.jsx:28]: I'm fairly certain that it will keep the state of value even through the rerender, not sure though.

**Review by BTanaka11 — APPROVED** [review]
Looks good. See my comments, which would make it so the browser console alerts don't show. Also, for formatting date, feel free to grab the function I'm using

File: client/src/components/Q&A/SearchBar.jsx (hunk 3 above)
Comment (BTanaka11) [reply client/src/components/Q&A/SearchBar.jsx:28]: I stand corrected. Gave it a test and yep, it was needed. Sorry about that!

**Pull request merged**